gradle/rewrite.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'org.openrewrite.rewrite'
 rewrite {
 	activeRecipe('com.diffplug.spotless.openrewrite.SanityCheck')
+	configFile = project.getRootProject().file("${rootDir}/gradle/rewrite.yml")
```

Comment:
also follow project convent, thus move away from root not to bother around.

same here.

kind of takes away the focus.